### PR TITLE
fix psalm build

### PR DESCRIPTION
**Pull request by azjezz**
| Q             | A
| ------------- | ---
| Bugfix?       | ✔️
| Breaks BC?    | ❌
| New feature?  | ❌ 
| Issues        | n/a
| Docs PR       | n/a

- fixed psalm build
- move to psalm level 7

### CODE REVIEW

File: src/Attributes/src/Internal/Instantiator/DoctrineInstantiator.php [hunk 1]
```diff
@@ -136,11 +135,6 @@ private function propertyNotFound(\ReflectionClass $attr, string $name, ?\Reflec
         return new SemanticAttributeException($message);
     }
 
-    /**
-     * @param \ReflectionClass $class
-     * @return string
-     */
-    #[Pure]
     private function getAvailablePropertiesString(\ReflectionClass $class): string
```
Comment (azjezz) [inline src/Attributes/src/Internal/Instantiator/DoctrineInstantiator.php:138]: this function is not pure.

Comment (azjezz) [reply src/Attributes/src/Internal/Instantiator/DoctrineInstantiator.php:138]: ( note: don't trust phpstorm when it says that a function is pure, phpstorm purity checker is completely broken )

Comment (SerafimArts) [reply src/Attributes/src/Internal/Instantiator/DoctrineInstantiator.php:138]: yea, thx!

**Comment by butschster** [comment]
@azjezz Hi, thank you for PR. I carefully studied it and made a new one #493 with some of changes from your PR. Most of changes from your PR were fixed in this PR #472

**Pull request closed without merging**